build.sh

```diff
@@ -435,13 +435,19 @@ build_CoreLib()
         exit 1
     fi
 
+    local __CoreLibILDir=$__BinDir/IL
+
     if [ $__SkipCrossgen == 1 ]; then
         echo "Skipping generating native image"
+
+        if [ $__CrossBuild == 1 ]; then
```

Comment:
This should be changed to armel specifically, we do not want to use the IL SPC for arm64 or armhf. It is unclear to me if we want the IL SPC for armel.